packages/desktop-client/e2e/page-models/rules-page.js

```diff
@@ -61,6 +61,29 @@ export class RulesPage {
         this.page.getByTestId('action-list'),
       );
     }
+
+    if (data.splits) {
+      let idx = 0;
+
+      if (data.splits.beforeSplitActions) {
+        idx = data.splits.beforeSplitActions.length;
```

Comment:
We can just put this inside the splitActions if block below

Reply:
Done 👍